backend/migrations/sqls/20241002111606-init-schema-up.sql

```diff
@@ -15,6 +15,13 @@ CREATE TABLE transaction_results (
   transaction_result jsonb,
   head_cid TEXT
 );
+-- Users
+CREATE TABLE users (
```

Comment:
Do we see a use case for keeping user email?
For example, when an Alice share a file with Bob, we could send an email notification to Bob, or if Bob request access to a file Carl has uploaded?

Reply:
Yes, it makes sense I thought about introducing other data such as full name, email, profile photo in other PR